lib/candig-data-portal/docker-compose.yml

```diff
@@ -27,6 +26,12 @@ services:
         - "traefik.http.routers.candig-data-portal.rule=Host(`candig-data-portal.${CANDIG_DOMAIN}`)"
         - "traefik.http.services.candig-data-portal.loadbalancer.server.port=${CANDIG_DATA_PORTAL_PORT}"
     logging: *default-logging
+    environment:
+      - REACT_APP_KATSU_API_SERVER=${TYK_KATSU_API_TARGET}
+      - REACT_APP_CANDIG_SERVER=${TYK_CANDIG_API_TARGET}
+        #- REACT_APP_FEDERATION_API_SERVER=http://${CANDIG_DOMAIN}:${FEDERATION_PORT}
+        #- REACT_APP_BASE_NAME='/'
+      - REACT_APP_SITE_LOCATION=UHN
```

Comment:
should this be a variable instead?

Reply:
What should its default value be? I just grabbed this change from the candig-dev deployment...

Reply:
I made a new variable, CANDIG_SITE_LOCATION, in example.env.